lib/router.js

```diff
@@ -7,6 +7,7 @@
 
 var debug = require('debug')('Routr:router');
 var pathToRegexp = require('path-to-regexp');
+var queryString = require('query-string');
```

Comment:
Add the ability to pass `options.queryLib` to the constructor to change out the query parsing/stringifying implementation.